test/test_core.py

```diff
@@ -5683,6 +5683,7 @@ def test_getdents64_special_cases(self):
   def test_getcwd_with_non_ascii_name(self):
     self.do_run_in_out_file_test('fs/test_getcwd_with_non_ascii_name.cpp')
 
+  @no_wasmfs('no support for /proc/self/fd/, see https://github.com/emscripten-core/emscripten/issues/19430')
```

Comment:
This is unrelated, right?  (no need to split it out, just wanted to confirm)

Reply:
Yes (there were explanations of these trivial additions in the previous PR that this replaces).